infra/cifuzz/cifuzz.py

```diff
@@ -109,10 +109,9 @@ def build_fuzzers(project_name,
 
   # Check that sanitizer is valid.
   if not is_project_sanitizer(sanitizer, project_name):
-    logging.info("%s is not a project sanitizer, defaulting to address.",
-                 sanitizer)
-    sanitizer = 'address'
-  logging.info("Using %s as sanitizer.", sanitizer)
+    logging.info("%s is not a project sanitizer.", sanitizer)
```

Comment:
logging.error ?

Reply:
The logging is redundant with the exception. Got rid of it.